sabnzbd/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: sabnzbd_web_1
+      APP_PORT: 8080
+
+  web:
+    image: lscr.io/linuxserver/sabnzbd:4.1.0@sha256:84a54b2bd29198bc76e253c8592ac72996737d79f845c1db56ab053739cb61bc
+    restart: unless-stopped
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/config
+      - ${UMBREL_ROOT}/data/storage/downloads:/downloads
```

Comment:
Nice, this looks good. The container is running as 1000:1000 so any directories created in `${UMBREL_ROOT}/data/storage/downloads` will have the proper permissions 👌.

### Some thoughts on initial set up

One thing I noticed when checking out the app is that the default downloads folders are set up like this:

<img width="1014" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/88b8cf58-fd84-4e4b-94ad-cf1557d3fa3f">

So in the container downloads are stored at `/config/Downloads/`. We bind `${APP_DATA_DIR}/data/config:/config`, so by default a user would be storing downloads on their Umbrel at `~/umbrel/app-data/sabnzbd/config/Downloads` instead of where we want them to be accessible at `${UMBREL_ROOT}/data/storage/downloads`. This means they wouldn't be able to access them via an app like File Explorer and would have to ssh in to their Umbrel to access them.

Unfortunately it doesn't look like we can pass environment variables to override this (I could be wrong). That leaves us a couple options. I'm not sure what's best and would love your thoughts:

1. We include instructions in the app description to change the download folders to `/downloads/complete` and `/downloads/incomplete` in the UI:

<img width="1138" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/3a610546-c24c-454f-b6f8-b7815bf4dec9">

2. We add script logic to edit the config so that by default `/downloads` inside the container is used by default

3. We change the bind mounts in such a way that we bind `/config/Downloads` in the container to the main downloads directory on the host.

Reply:
Hey @nmfretz , I just quickly went over it and discovered that this is actually an Issue.

https://github.com/linuxserver/docker-sabnzbd/issues/71#issuecomment-681632799

When the Sabnzbd wizard runs it seems to change / modify the `sabnzbd.ini` file. So I don't think we can create a PostInstal script to change the paths. 

I haven't looked at option 3 yet, I'll look into it.

Reply:
Hey @nmfretz, I fixed this issue by the 3rd option you mentioned. 

> We change the bind mounts in such a way that we bind /config/Downloads in the container to the main downloads directory on the host.

Reply:
Nice. Ya, I like this option as well as I think it is the one with the least overhead for us moving forward. We don't have to worry about the structure of the sabnzbd.ini file changing in the future like we would with option 2. And I like it better than Option 1 because we reduce the amount of set up that a user has to do manually.

> Hey @nmfretz , I just quickly went over it and discovered that this is actually an Issue.

Oh ya, good find. The issue actually links to the README where they say this is the desired behaviour:
https://github.com/linuxserver/docker-sabnzbd#application-setup

We still have to make a change to your current bind mounts:
```yaml
volumes:
      - ${APP_DATA_DIR}/data/config:/config
      - ${UMBREL_ROOT}/data/storage/downloads:/config/Downloads
      - ${UMBREL_ROOT}/data/storage/downloads/incomplete:/config/incomplete-downloads
```

The default location that the set-up wizard uses for incomplete downloads is `/config/Downloads/incomplete` instead of `/config/incomplete-downloads`. That being said, I think we should just bind `/config/Downloads` and we will get both complete and incomplete. So we would do:

```yaml
volumes:
      - ${APP_DATA_DIR}/data/config:/config
      - ${UMBREL_ROOT}/data/storage/downloads:/config/Downloads
```

<img width="1014" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/88b8cf58-fd84-4e4b-94ad-cf1557d3fa3f">

Reply:
Ah right, My bad on that one. I've removed the incomplete downloads mount and only included the downloads dir.